HW2.js

```diff
@@ -0,0 +1,9 @@
+var numArr = process.argv;
```

Comment:
Since you are declaring this here, you should do one of two different things here
1. set numArr to process.argv.length - 1, so you don't have to do that in your for loop
or
2. keep it as it is, but in your for loop use `Number(numArr[pos])`